code_files/main.py

```diff
@@ -0,0 +1,33 @@
+import argparse
```

Comment:
`main.py` --> `weatherman.py `

Reply:
[See this latest commit](https://github.com/ttqureshi/WeatherMan/commit/a9bfba76030779f62875a46887178ea2565ca521)